provision/vvv-init.sh

```diff
@@ -183,30 +192,36 @@ update_wp() {
   fi
 }
 
-setup_database
-setup_nginx_folders
-
-cd "${VVV_PATH_TO_SITE}/public_html"
+setup_cli() {
```

Comment:
we should be explicit about the path this file is in, in future we'll want to consider using a template system similar to the nginx config so users can add other things in here, but not in this PR